src/main/java/ChooseFile.java

```diff
@@ -15,7 +15,9 @@ public void start(Stage primaryStage) throws Exception{
         FXMLLoader fxmlLoader = new FXMLLoader(getClass().getResource("/ChooseFile.fxml"));
         Parent root = (Parent) fxmlLoader.load();
         ((ChooseFileController) fxmlLoader.getController()).setStage(primaryStage);
-        primaryStage.setResizable(false);
+        primaryStage.setResizable(true);
```

Comment:
Is it automatically set to true or is this needed for it to become resizable? I only had it set to false cause it was breaking when I would try to resize.

Reply:
Not sure, I didn't check it without anything, I just assumed since it was set to false I'd set it to true and see